pydatastructs/linear_data_structures/arrays.py

```diff
@@ -416,6 +416,7 @@ def delete(self, idx):
             self._num -= 1
             if self._last_pos_filled == idx:
                 self._last_pos_filled -= 1
+                # self._size -= 1 # Check if size of array should be changed
```

Comment:
This needed now?

Reply:
This fixes the bug for the code here: https://github.com/codezonediitj/pydatastructs/issues/557, but other cases need to be tested.

Reply:
I'm removing it for now.